<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutProgram</class>
 <widget class="QWidget" name="AboutProgram">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>530</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>770</width>
    <height>530</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>770</width>
    <height>530</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>A B O U T   P R O G R A M</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(58, 37, 179);
</string>
  </property>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ABOUT PROGRAM</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_line1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>581</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>line_1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_line2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>581</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>line_2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_line3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>581</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>line_3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_line4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>310</y>
     <width>581</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>line_4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>450</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton#close:hover{
    background-color: rgb(99, 199, 147);
};

background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
